### ISSUE TITLE
Is this the official repository for the ACNN paper?

### ISSUE THREAD

**Issue by normandra** [XiaoYunZhou27/ACNN#1]
Hi I just read [ACNN's paper](https://arxiv.org/pdf/1901.09203.pdf). It was mentioned that the corresponding code is put up on github. Is it still going to be pushed?

**Comment by SuzyZhou27** [comment]
Yes. It is. But due to the authors have graduated. It is no longer continously maintained and updated.
